### DISCUSSION TITLE
Multiple Keys

### DISCUSSION THREAD

**Discussion by jaikishantulswani** [projectdiscovery/uncover#160] [Q&A] [answered]
Can we use multiple keys from different account of similar service

**Comment by forgedhallpass** [comment, marked as answer]
It's in the [README](https://github.com/projectdiscovery/uncover#provider-configuration):

> * Multiple API key input support
> * Automatic API key randomization  
> * When multiple keys/credentials are specified for same provider in the config file, random key will be used for each execution.
